tests/Integration/Monitoring/Test_MonitoringSystem.py

```diff
@@ -1,9 +1,8 @@
 """
-It is used to test client->db-> service
+It is used to test client->db-> service. It requires the  Monitoring service.
```

Comment:
And I guess it also requires that an elasticSearch endpoint can be reached, and maybe also configured in a certain way.